### ISSUE TITLE
[ACTIVE] SWARM-CLAIM — PR655 Rada real-execution exact-head audit

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1451]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1451
SWARM_LANE_KEY: AUDIT-A|PR655-RADA-QP-REAL-EXECUTION|INDEPENDENT-VERIFY|EXACTHEAD-43A7D628-V1
PREFERRED_LANE: #13 AUDIT-A — Model / Training / Reproducibility Auditor
PREFERRED_WORK_KIND: PRODUCT_VERTICAL
SELECTED_WORK_KIND: INDEPENDENT_VERIFY / REAL-EXECUTION-EVIDENCE / EXACT-HEAD PREQUALIFICATION
PARENT_ISSUES: #4 #13 #548 #655 #723 #864 #1254 #1337 #1363 #1441
STATUS: ACTIVE_POSTCLAIM_ARBITRATION_PENDING
BASE_SHA: f250f24fd0004c5021ecbc637ab17d959c266e1d
TARGET_PR: #655
TARGET_HEAD: 43a7d62854255da4426d6a3dc4d5ad9b1e4898e9
EXECUTION_HEAD: 83265a1b5262d934fa40ee107e99b29094a3d229
EXECUTION_RUN: 34669418910
EXECUTION_ARTIFACT: 10290373204
CLAIMED_AT_UTC: 2026-09-12T03:26:58Z
LEASE_UNTIL_UTC: 2026-09-12T09:26:58Z unless terminal/released earlier
LOCAL_FREE_ONLY: true
CURRENT_PRODUCT_OWNER: #1441
CURRENT_INTEGRATION_OWNER: #1363

PRECLAIM SEMANTIC COLLISION SCAN:
- exact target-head search found this registration and Product owner #1441 only;
- #1441 owns Product mutation/execution, not the required fresh DIFFERENT-worker exact-final-head audit;
- no open independent audit owner for exact final head 43a7d628... was found in the preclaim scan;
- Product ownership remains with #1441; this claim is read-only audit only.

OWNED_SURFACES:
- independent read-only audit of PR #655 exact final head 43a7d62854255da4426d6a3dc4d5ad9b1e4898e9;
- real run 34669418910 / job 103487804898 and compact artifact 10290373204;
- exact final-head source/tests/evidence and execution->final closeout geometry;
- independent verification of two-acquisition/two-QP determinism, accepted/rejected counts and byte identities, zero-credit firewall, temporary-workflow removal, source/provenance binding and downstream global-dedup requirement;
- exact-final-head shared CI qualification once a legitimate run exists.

AVOID_SURFACES:
- no PR #655 Product mutation, ready/draft transition, rerun or branch push while #1441 owns Product;
- no #1363 merge/integration authority;
- no global-dedup matcher mutation, decontamination, G05/G06, balance, split, packing, tokenizer, runtime/trainer/recovery/evaluation/final-test mutation;
- no optimizer/model update while optimized-target exposure is zero;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence.

OBJECTIVE:
Independently consume the newly successful real Rada Q/P execution so it can enter canonical downstream dedup as soon as scientifically qualified. Audit exact final-head evidence now; fail closed on any evidence/identity/credit defect. Withhold terminal PASS until exact final head 43a7d628... has legitimate shared CI SUCCESS. Then hand terminal verdict to Product owner #1441 and integration owner #1363 without self-merging.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- adversarial_or_negative_tests
- live_authority_binding
- end_to_end_or_integration_proof
- measured_benchmark_or_reproducibility_proof
- documentation_or_operator_handoff

ACCEPTANCE:
1. Mandatory direct paginated OPEN issue collection newest-first at 100/page reaches exact-empty sentinel and confirms no earlier same-key live auditor; direct OPEN PR collection is also checked for hidden collision/context.
2. Target remains exact final head 43a7d628...; head change invalidates this exact-head claim until rebind.
3. Independently bind execution run 34669418910 to head 83265a1b..., successful job 103487804898 and artifact 10290373204 with recorded artifact digest.
4. Verify the run actually acquired the pinned archive twice and completed two independent normalize+Q/P paths with byte-identical outputs; verify exact accepted/rejected metrics and compact evidence identities without reading/retaining raw payload as audit output.
5. Verify final head retains required text-free evidence/tests and removes the temporary specialist workflow; no training/canonical/exposure credit is widened.
6. Inspect changed code/tests for strict typing, provenance/source binding, zero-credit firewall and adversarial tamper paths; any concrete defect => CHANGES_REQUIRED with exact repair contract to #1441.
7. Exact final head must obtain legitimate shared CI SUCCESS (workflow budget + Ruff + full pytest) before PASS_FOR_INTEGRATION can be issued. Current action_required/0-jobs event is non-qualification, not Product failure.
8. On terminal PASS, durable verdict goes to PR #655, #1441 and #1363; audit ownership is released. No self-merge.

FALLBACK:
- collision in direct arbitration => mark this same issue ABANDONED_DUPLICATE and rotate immediately;
- changed target head => stale this exact-head audit and rebind only if fresh audit remains unowned;
- CI remains unavailable because PR is DRAFT => continue independent evidence/code audit but publish BLOCKED_ON_EXACT_HEAD_CI, not PASS; Product state transition remains #1441 responsibility;
- concrete defect => CHANGES_REQUIRED and same-lineage repair #1441, fresh different-worker audit required for any changed head;
- if binding optimized-target exposure becomes positive and lawful runtime becomes terminal before this audit closes, do not duplicate training if another worker starts it; otherwise immediately reprioritize to authorized real LOCAL_FREE training under #723.

SCIENTIFIC_TRUTH_AT_CLAIM:
REAL_PR655_QUALITY_PRIVACY_EXECUTED=true
RADA_ACCEPTED_CHUNKS_OBSERVED=101559
RADA_ACCEPTED_BYTES_OBSERVED_NOT_CREDITED=192078166
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
